src/java/com/unifina/signalpath/blockchain/Web3jHelper.java

```diff
@@ -323,6 +324,29 @@ public static Type web3jArrayGet(Array array,int... indices){
 		return (Type) val;
 	}
 
+	public static Web3j getWeb3jConnectionFromConfig(){
+		EthereumModuleOptions ethereumOptions = new EthereumModuleOptions();
```

Comment:
Feels like `EthereumModuleOptions` should use a helper, not the other way around! (`EthereumModuleOptions` is much more specific.)

Reply:
That same can be accomplished by either duplicating the code fetching defaultNetwork and then the URL, or maybe refactoring out the static method of getting the defaultNetwork's URL. But this function does what the name says: gets web3j connection corresponding to the default network of the E&E installation